tuned/plugins/plugin_scheduler.py

```diff
@@ -584,8 +584,10 @@ def _set_irq_affinity(self, irq, affinity, restoring):
 				log.debug("Setting SMP affinity of IRQ %s is not supported"
 						% irq)
 			else:
-				log.error("Failed to set SMP affinity of IRQ %s to '%s': %s"
-						% (irq, affinity_hex, e))
+				# managed IRQs would have smp_affinity file as RO, so
+				# it could be fine
+				log.warn("Failed to set SMP affinity of IRQ %s to '%s': %s"
+						 % (irq, affinity_hex, e))
```

Comment:
I think I'll wait for a resolution in our email thread regarding whether Tuned should manage IRQ affinities at all before digging into this :).